oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/IndexOptions.java

```diff
@@ -102,6 +103,8 @@ public IndexOptions(OptionParser parser){
         //Set of options which define action
         actionOpts = ImmutableSet.of(stats, definitions, consistencyCheck, dumpIndex, reindex, importIndex);
         operationNames = collectionOperationNames(actionOpts);
+        existingDataDumpDirOpt = parser.accepts("existing-data-dump-dir", "Directory containing mongo document dumps from previous incomplete run")
```

Comment:
I would avoid the term "mongo document" and use "document store". For multiple reasons: the dumps don't contain any MongoDB documents actually, but Oak data. Then, at some point we might also support RDBMS (I know currently it needs MongoDocumentStore - but that's an implementation detail)

Reply:
done